### PR TITLE
Read file

### PR DESCRIPTION
**Pull request by cyrusHwang**
@AlexEdgcomb

### CODE REVIEW

**Comment by cyrusHwang** [comment]
@AlexEdgcomb

File: progression-csv-downloader.py [hunk 1]
```diff
@@ -3,29 +3,33 @@
 import json
 
 directoryName = input("Enter path to destination folder: ")
+#directoryName = "C:/Users/hwang/Programming/Work/progression_activities_analysis/UCRCS10Spring2016_progression_activities_csvs"
```
Comment (AlexEdgcomb) [inline progression-csv-downloader.py]: Please remove this line of code. If you ever need, you can look at the old commits to find this line of code. That's the power of a code repository!

File: progression-csv-downloader.py [hunk 2]
```diff
@@ -3,29 +3,33 @@
...
+    
 while True:
+    authToken = input("Enter auth token: ").strip()
+    zybookCode = input("Enter zybook code: ").strip()
+    #zybookCode = "UCRCS10Spring2016"
+    
+    choice = input("Enter resource IDs manually (1) or enter text file with resource IDs (2): ")
+    if choice == '1':
+        contentIdsStr = input("Enter content IDs separated by spaces: ").strip()
+        #contentIdsStr = "3648060 3648065"
```
Comment (AlexEdgcomb) [inline progression-csv-downloader.py]: Please remove this line of code

File: progression-csv-downloader.py [hunk 3]
```diff
@@ -3,29 +3,33 @@
...
+        contentIdsList = contentIdsStr.split(' ')
+        print(contentIdsList)
+    elif choice == '2':
+        pathToFile = input("Enter path to text file: ")
+        #pathToFile = "C:/Users/hwang/Programming/Work/progression_activities_analysis/Resource_IDs_Prog_CA_UCRCS10Spring2016.txt"
```
Comment (AlexEdgcomb) [inline progression-csv-downloader.py]: Please remove this line of code too

File: progression-csv-downloader.py [hunk 4]
```diff
@@ -3,29 +3,33 @@
...
+        for contentID in contentIdsList:
+            responseURL = "https://s2.zybooks.com/v1/zybook/{}/submissions/{}?auth_token={}".format(zybookCode, contentID, authToken)
+            urlContentText = urllib.request.urlopen(responseURL)
+            urlContentJsonObj = json.loads(urlContentText.read().decode('utf-8-sig'))
+            csvURL = urlContentJsonObj["url"]    
+            urllib.request.urlretrieve(csvURL, filename="{}/{}_{}.csv".format(directoryName, contentID, zybookCode))
         break;
     except:
-        print("Resource was not found at {}. Try again.".format(responseURL))
+        print("Invalid URL. Try again.")
```
Comment (AlexEdgcomb) [inline progression-csv-downloader.py]: Perhaps print out the bad URL, or at least the error?

**Comment by AlexEdgcomb** [comment]
@cyrusHwang , looks good. I left a few comments.

**Comment by cyrusHwang** [comment]
@AlexEdgcomb ready for review

**Comment by AlexEdgcomb** [comment]
@cyrusHwang , merged. Nice work.

**Pull request merged**